### PR TITLE
Rename chimera-controller to kubewarde-controller

### PR DESCRIPTION
**Pull request by flavio**
Also ensure the new CRDs are in place

### CODE REVIEW

File: charts/kubewarden-controller/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kubewarden-controller installed.
+
+You can start defining cluster admission policies by using the
+`aclusteradmissionpolicies.policies.kubewarden.io` resource.
```
Comment (ereslibre) [inline charts/kubewarden-controller/templates/NOTES.txt:4]:
```suggestion
`clusteradmissionpolicies.policies.kubewarden.io` resource.
```

File: charts/kubewarden-controller/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+For more information checkout https://kubewarden.github.io/
```
Comment (ereslibre) [inline charts/kubewarden-controller/templates/NOTES.txt:6]:
```suggestion
For more information checkout https://kubewarden.io/
```

**Review by ereslibre — CHANGES_REQUESTED** [review]
A couple of changes, LGTM afterwards, thanks @flavio

**Comment by flavio** [comment]
Thanks for the fixes!

**Pull request merged**